src/mono/wasm/runtime/driver.c

```diff
@@ -470,6 +510,7 @@ mono_wasm_load_runtime (const char *unused, int debug_level)
 		mono_register_bundled_assemblies ((const MonoBundledAssembly **)bundle_array);
 	}
 
+	mono_wasm_register_bundled_satellite_assemblies ();
```

Comment:
wasn't sure if we want to call it from here or from library_mono.js. We need this for our tests and for other scenarios when satellite assemblies are already added to the array before we load the runtime.

Reply:
I think this should be fine.